problem_builder/tests/integration/test_titles.py

```diff
@@ -72,7 +72,7 @@ class StepTitlesTest(SeleniumXBlockTest):
     )
 
     mcq_template = """
-        <problem-builder mode="{{mode}}">
```

Comment:
Yeah, I think these were originally copied from a django template. Not sure why they worked before this change.

Incidentally, the `mode` parameter is deprecated, and we will eventually remove support for it.